Mathlib/Tactic/Common.lean

```diff
@@ -11,6 +11,14 @@ import Qq
 -- Tools for analysing imports, like `#find_home`, `#minimize_imports`, ...
 import ImportGraph.Imports
 
+-- Currently we don't need to import all of ProofWidgets,
+-- but without this, if you don't run `lake build ProofWidgets` then `make test` will fail.
+-- Hopefully `lake` will be able to handle tests later.
+import ProofWidgets
```

Comment:
Why did this come back?

Reply:
I think that [this comment](https://github.com/leanprover-community/mathlib4/pull/11807#discussion_r1554644432) made me add it.  I suspect that it was there at the time that this PR was opened, then the other PR was merged, the import possibly removed, but when this PR was looked at again, no one noticed that it was still here.